### PR TITLE
Updated Hardware Deployment

### PR DESCRIPTION
**Pull request by normandmickey**
The installation process for Ubuntu is very similar to the RPI now.  Updated the hardware and installation instructions.

### CODE REVIEW

File: docs/Deployment/LightningInABox.md [hunk 1]
```diff
@@ -1,145 +1,209 @@
...
-Here's another Bitcoin axiom "Your hardware. Your node. Your keys. Your Bitcoin".
+[Lightning in a Box](https://lightninginabox.co)
+"Your hardware. Your node. Your keys. Your Bitcoin".
+
 
 In order for your Bitcoin experience to be truly self-sovereign and trustless you should consider **running nodes on your own hardware** and internet connection.
 **BTCPay Server is an excellent way to run both Bitcoin & Lightning nodes**.
 Not only are you validating transactions you also get the ability to accept base layer Bitcoin and second layer Lightning payments.
-To that end here are instructions to install and host your very own BTCPay Server.
+Followng are instructions to install and host your very own BTCPay Server.
```
Comment (dennisreimann) [inline docs/Deployment/LightningInABox.md:10]:
```suggestion
Following are instructions to install and host your very own BTCPay Server.
```

File: docs/Deployment/LightningInABox.md [hunk 2]
```diff
@@ -1,145 +1,209 @@
...
-For the sake of speed, compatibility and ease of installation it is recommended that you use an Intel/AMD based computer with a minimum of 4 GB RAM and a 500 GB hard drive. The Gigabyte Brix or Intel NUC lines of compact computers are a great place to start. They are basically small, fanless boxes that are built using laptop components.
-
-**BTCPayServer can be successfully installed on the following hardware**. The end result is a small, quiet node that is fast enough to sync from block zero. The total cost is approximately $225 (including tax and delivery). These are "off the shelf" components that can be purchased from any online retailer (TigerDirect, NewEgg, Amazon, BestBuy etc). It will take roughly 24 hours to sync the Bitcoin blockchain. You can save little money by cutting the RAM and hard drive size in half.
+**BTCPayServer can be installed on the following hardware**. The end result is a small, quiet node that is fast enough to sync from block zero. The total cost is approximately $300.
```
Comment (dennisreimann) [inline docs/Deployment/LightningInABox.md:18]:
```suggestion
**BTCPay Server can be installed on the following hardware**. The end result is a small, quiet node that is fast enough to sync from block zero. The total cost is approximately $300. 
```

File: docs/Deployment/LightningInABox.md [hunk 3]
```diff
@@ -1,145 +1,209 @@
...
+1. [BeeLink S12 - Mini PC - $169.00](https://www.amazon.com/dp/B0C89TQ1YF?ref=nb_sb_ss_w_as-reorder-t1_k0_1_4&amp=&crid=SHKYOXZIRAO0&amp=&sprefix=beel)
+3. [WD Blue 2TB SSD - $129.00](https://www.amazon.com/Western-Digital-SA510-Internal-Solid/dp/B0C14TF467/ref=sr_1_3?crid=2WDY52E7ESSEB&dib=eyJ2IjoiMSJ9.MBxkb5ZIvwjKXOzscB0GUvsbhX1rVhilXNFzID6n0xHORsDBPkIxQhIixVuiLY9I16rlFs5COExAAD8761Do-tzuAnZiutbqN-KM9rAL4zCw94kA_ArCJeR_RTDynZbiXf2Phnahw1Gw2dqXVek3p0dpe6_a_fbJrqx4BRaieoYo0zj1mX6YPGaYZAmF2Vf_Quk1TrkARk6s1_wZ0vFUw7EWdjKJ9hmNLxPWMfADML90A1rXk8gSCcRnwV2jdzN7jCfg2_urfJZ3IWOW5X3iwnP7s-vSec88PGmQ3RhS-Rc.sEURveFhiTAHYwZQdwyJX72hpWL5UgD_3tEPet747oE&dib_tag=se&keywords=2tb+ssd+wd+blue&qid=1710685725&s=electronics&sprefix=2tb+ssd+wd+blue%2Celectronics%2C90&sr=1-3)
 
 Other requirements are as follows:
 
 1. High speed internet connection.
 2. Static IP
 3. Domain Name
-4. Ability to open ports (80, 443, 9735) on your router.
+4. Ability to open ports (80, 443, 9735) on your router (optional, BTCPayServer can be accessed over TOR or by Dynamic DNS).
```
Comment (dennisreimann) [inline docs/Deployment/LightningInABox.md:28]:
```suggestion
4. Ability to open ports (80, 443, 9735) on your router (optional, BTCPay Server can be accessed via Tor or by Dynamic DNS). 
```

File: docs/Deployment/LightningInABox.md [hunk 4]
```diff
@@ -1,145 +1,209 @@
...
-- `. ./btcpay-setup.sh -i`
-- `exit`
-
-**Step 11**
-Go to https://btcpay.yourdomain.com and confirm that your nodes are syncing.
-Enjoy!
-
-If you don't have the time or patience to build your own BTCPB there are a few merchants who can build one for you.
+### Install Fail2ban and GIT. OpenSSH server allows you to connect to your server using SSH clients (ie. [Putty](https://www.putty.org/)) 
+Fail2ban bans IP's that attempt to connect to your server and show malicious signs. GIT allows you to clone and manage repositories on github.com.
```
Comment (dennisreimann) [inline docs/Deployment/LightningInABox.md:65]:
```suggestion
Fail2ban bans IP's that attempt to connect to your server and show malicious signs.
Git allows you to clone and manage repositories on github.com.
```

File: docs/Deployment/LightningInABox.md [hunk 5]
```diff
@@ -1,145 +1,209 @@
...
-
-- `. ./btcpay-setup.sh -i`
-- `exit`
-
-**Step 11**
-Go to https://btcpay.yourdomain.com and confirm that your nodes are syncing.
-Enjoy!
-
-If you don't have the time or patience to build your own BTCPB there are a few merchants who can build one for you.
+### Install Fail2ban and GIT. OpenSSH server allows you to connect to your server using SSH clients (ie. [Putty](https://www.putty.org/))
```
Comment (dennisreimann) [inline docs/Deployment/LightningInABox.md:64]:
```suggestion
### Install Fail2ban and Git

OpenSSH server allows you to connect to your server using SSH clients (ie. [Putty](https://www.putty.org/)) 
```

**Review by dennisreimann — COMMENTED** [review]
If I see it correctly, the part from "Configuring the firewall" on is shared with the Raspberry Pi deployment docs, correct? If so we can maybe extract that part and include it in both docs, so that it will be easier to maintain.

**Comment by normandmickey** [comment]
The firewall and secondary drive instructions are the same now between the Raspberry Pi and mini PC install. As long as your mini PC has a boot drive and a second SSD. Not sure how to abstract those parts of instructions.  I have updated the LIAB instructions to match the RPI install for both sections.

**Comment by dennisreimann** [comment]
Ok, if you're done with the changes we can merge it as is. I can see later on how to maybe extract and share that part.

**Review by dennisreimann — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]

**Pull request merged**